### ISSUE TITLE
error on `npm start`

### ISSUE THREAD

**Issue by jchris** [sw-yx/netlify-fauna-todo#1]
I tried updating react but still got this issue maybe to do with   `<ConcurrentMode><StrictMode>`

<img width="1017" alt="screen shot 2018-12-30 at 11 31 23 am" src="https://user-images.githubusercontent.com/253/50550489-96372e00-0c26-11e9-80a4-56e0ae67b231.png">

Tried removing those, got:

<img width="1035" alt="screen shot 2018-12-30 at 11 34 34 am" src="https://user-images.githubusercontent.com/253/50550524-f4641100-0c26-11e9-95a4-28e3c7dc4559.png">

I'm on node v10.8.0

**Comment by swyxio** [comment]
thanks for filing an issue. you tried updating react? to what version? note i was on an alpha release here as i am using react hooks

**Comment by jchris** [comment]
I tried falling back to the non beta react, but it didn't help, because it doesn't have the feature. I have tried again with a clean checkout and still get the same issues. 

The best I can guess is somehow I'm not getting the React beta functionality, but that doesn't make sense to me. Double checking my node_modules reveals that the expected `react@^16.7.0-alpha.2` is in place.

Is anyone else able to reproduce this? The next step I can think of is to update my node install, I'll do that and report back.

**Comment by jchris** [comment]
Tried all the upgrades, still got the errors. In the `react` module I see this code which makes it hard to know where those features would come from.

```js
var enableStableConcurrentModeAPIs = false;
var enableHooks = false;

if (enableStableConcurrentModeAPIs) {
  React.ConcurrentMode = REACT_CONCURRENT_MODE_TYPE;
  React.Profiler = REACT_PROFILER_TYPE;
  React.unstable_ConcurrentMode = undefined;
  React.unstable_Profiler = undefined;
}

if (enableHooks) {
  React.useCallback = useCallback;
  React.useContext = useContext;
  React.useEffect = useEffect;
  React.useImperativeMethods = useImperativeMethods;
  React.useLayoutEffect = useLayoutEffect;
  React.useMemo = useMemo;
  React.useReducer = useReducer;
  React.useRef = useRef;
  React.useState = useState;
}
```

Maybe this means I need to hand edit my installed module? That doesn't seem right! Onwards...

**Comment by jchris** [comment]
I found a Stack Overflow answer that suggests that the issue is the `^` in the react package beta version. Apparently using that makes it not install the beta... I removed it and reinstalled and things are working now (locally) but I'll need to do another pass to make sure which changes are necessary, then I'll send a PR.

**Comment by swyxio** [comment]
yea 16.7.0 was released after 16.7.0-alpha.2 which probably caused it. lesson learned... thanks for figuring it out!
